src/applications/caregivers/components/PreSubmitInfo/SignatureCheckbox.jsx

```diff
@@ -49,7 +50,10 @@ const SignatureCheckbox = ({
         />
 
         {isRepresentative && (
-          <p className="on-behalf-representative">
+          <p
+            className="on-behalf-representative"
+            id={`${label}-signature-label`}
+          >
```

Comment:
I implemented this code but will not work till the feature I noted in the ticket is completed on the DSVA global component